src/screens/HyperSwitch/SDKPayment/SDKPage.res

```diff
@@ -128,6 +134,17 @@ let make = () => {
     Js.Nullable.null->Promise.resolve
   }
 
+  React.useEffect0(() => {
+    let sdkBaseUrl = switch HSwitchGlobalVars.hostType {
+    | Live => "https://checkout.hyperswitch.io/v0/HyperLoader.js"
```

Comment:
rather than defining the url directly we should take from the env value for the sdk url